HISTORY.md

```diff
@@ -1,5 +1,18 @@
 # Release History - `Contribution Service`
 
+# 0.2.0 (2023-01-25)
+
+- Improved test coverage
+- Improved handler routing implementation
+- Added release workflow
+- Added attributes to the token metadata
+- Added layer threshold format during API data validation
+- Changes handler health implementation to get proper healthcheck even if Tendermint dies
+- Corrected how the seconds until next reset is calculated for the healthcheck
+- Handle exceptions when the service account does not have write permissions.
+- Added initial documentation to the repository
```

Comment:
We are missing the bump: we now use open autonomy 0.8. Also, once merged please tag it.